packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -224,6 +224,12 @@ const Banner = (props: Props): React.Node => {
             </View>
         </View>
     );
+
+    return layout === "full-width" ? (
+        bannerBody
+    ) : (
+        <View style={styles.containerFloating}>{bannerBody}</View>
+    );
```

Comment:
I was thinking of just adding a container around the whole thing instead of only in the floating case, but that required adding two different styles and switching between them depending on layout. I think that was also a completely find way of doing this, and it's not a big change if people would prefer using that method instead.